x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/config_panel.test.tsx

```diff
@@ -147,7 +171,11 @@ describe('ConfigPanel', () => {
         first: mockDatasource.publicAPIMock,
         second: mockDatasource.publicAPIMock,
       };
-      const component = mountWithIntl(<LayerPanels {...defaultProps} />);
+      const component = mountWithIntl(
+        <LayerPanels {...defaultProps} />,
+        // @ts-expect-error
```

Comment:
Why expecting an error here? It seems like we can just extend the options type in `packages/kbn-test/src/jest/utils/enzyme_helpers.tsx` to take `attachTo` as well.